src/components/pages/TeamPage.jsx

```diff
@@ -0,0 +1,126 @@
+
+import React from 'react';
+import GreenBlobImage from '../../assets/blob_green.png';
+import PurpleBlobImage from '../../assets/blob_dusty_purple2.png';
+import YellowBlobImage from '../../assets/blob_yellow.png';
+import teamMembers from '../../data/teamData';
+import ScrollToTop from '../layouts/ScrollToTop';
+
+
+const TeamPage = () => {
```

Comment:
Great choice to contain the team data in another module and importing all that info in an array.